### ISSUE TITLE
[SUGGESTION] | undead/demonic status reversion

### ISSUE THREAD

**Issue by virsia** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth#785]
**DO NOT REMOVE PRE-EXISTING LINES**
----------------------------------------------------------------------------------------------------------
**Describe your suggestion in full detail below:**

need way to reverse undead/demon status since there is none, due to it's mechanic undead and demonic character cannot produce heir using natural means due massive fertility malus even though they changing religion to non death god or non fel religion and their adopted children always be undead/demon. 

suggest for author to make rare event to deal with this issue to remove demonic/undead trait. 

how to reproduce this? 
1. your only child get captured by lich king and he decide to make him/her zombie, adoption always result in zombie character heir.
2. play on second start date and gets undead plague while you have no living being heir, adoption always result in zombie character heir.
3. prolong fight in fel territory having fel corruption raised to 100 and become demon this can be avoided by resign as commander.
4. starting as demonic character but tired being demon want to play as succubus but because no seduction no one being cuckcolded feels kinda bland because demon supposedly full of debauchery and sins change religion and start fighting for the good guys adopted child still a demon.

need zombie/demon romance because some zombie/demon have very warm heart.

**Comment by zumbak04** [comment]
> suggest for author to make rare event to deal with this issue to remove demonic/undead trait.

The lore says it's impossible. I suggest another thing: chance to adopt normal living heir.

**Comment by virsia** [comment]
> > suggest for author to make rare event to deal with this issue to remove demonic/undead trait.
> 
> The lore says it's impossible. I suggest another thing: chance to adopt normal living heir.

lorewise it is impossible to reverse being undead but still and adoption sure it's one way for zombie to get's living heir

now how about demon? demon basically alive but still due to massive infertility malus and being a demon make's you being hated like- 40, I understand as to why Undead have fertility malus they are dead but I don't understand as to why demon also limited in having to procreate. why not just limit them like how many children they can have at a time, in CK2 picking seduction eventually will give you nice opinion modifier or if you could get lucky you can get seducer trait which can be used to partially negate opinion malus (-40) for being demon.

undead romance or event to make undead alive again is just roleplay purpose undead in WOW is a soul being enslaved by necromancer basically if necromancer lose control they are gaining self consciousness.

in WOW we could could create golem or making abomination why not try to create homunculus for roleplay purposes in this mod, I suggest an event so while there is possibility for undead to be alive again it is not as easy and simple as to pick living heir adoption. basically just transfer soul to artificial living body the mechanics is actually the same as why undead is there at the first place I mainly suggest this for roleplay purposes so mods still have that rewards and punishment aspect.

**Comment by zumbak04** [comment]
> now how about demon? demon basically alive but still due to massive infertility malus and being a demon make's you being hated like- 40, I understand as to why Undead have fertility malus they are dead but I don't understand as to why demon also limited in having to procreate. why not just limit them like how many children they can have at a time, in CK2 picking seduction eventually will give you nice opinion modifier or if you could get lucky you can get seducer trait which can be used to partially negate opinion malus (-40) for being demon.

I don't know demons that can reproduce... maybe except the aranasi. Demon souls are immortal, so there's no need in it.

> in WOW we could could create golem or making abomination why not try to create homunculus for roleplay purposes in this mod, I suggest an event so while there is possibility for undead to be alive again it is not as easy and simple as to pick living heir adoption. basically just transfer soul to artificial living body the mechanics is actually the same as why undead is there at the first place I mainly suggest this for roleplay purposes while it is not lore friendly it is not easy problem to solve and mods still have that rewards and punishment aspect.

I'm not familiar with such examples in the lore. Abominations and homunculi are classified as undead. So, I think there's no way to transfer the soul without breaking the body.
https://wow.gamepedia.com/Abomination
https://wow.gamepedia.com/Homunculus

**Comment by virsia** [comment]
> I don't know demons that can reproduce... maybe except the aranasi. Demon souls are immortal, so there's no need in it.
 
Having "no need" to reproduce is different with "cannot" reproduce.

Having no need means they "choose" to not to reproduce while they can while cannot reproduce is barred from any luxury or choice for that matter, Mortal who deal with demons have many different wish not all of them desire power or wealth some desire other thing, immortal is not invincible after all, beside if author classify demon as the same why make them able to change their way if they are having kind trait? 

> I'm not familiar with such examples in the lore. Abominations and homunculi are classified as undead. So, I think there's no way to transfer the soul without breaking the body.

Homunculi able to preserve partial of their body this artificial body is so much resemblance of human being that demon hunter unable to distinguish them, if lore stated that they are able to preserve and making something pretty much alive with meager magic or any technique, in theory they are able to do more with more resources and time invested in it, with proper experiment of course.
So basically if some parts of the body made active again procreation is actually viable even making fully active body is possible given enough time and resources invested.

the thing is in ck2 we are basically in open world with many possibility the drive to achieve something despite the odds against it is the main drive to pushing forward in the game. like immortality quest it is not even real in real life role yet they add it to give the game some flavor they give that what if scenario.

**Comment by zumbak04** [comment]
> Having "no need" to reproduce is different with "cannot" reproduce.
> 
> Having no need means they "choose" to not to reproduce while they can while cannot reproduce is barred from any luxury or choice for that matter, Mortal who deal with demons have many different wish not all of them desire power or wealth some desire other thing, immortal is not invincible after all, beside if author classify demon as the same why make them able to change their way if they are having kind trait?

What way? If you mean reproduction, as I said, we don't see lore examples except one race.

> Homunculi able to preserve partial of their body this artificial body is so much resemblance of human being that demon hunter unable to distinguish them, if lore stated that they are able to preserve and making something pretty much alive with meager magic or any technique, in theory they are able to do more with more resources and time invested in it, with proper experiment of course.
> So basically if some parts of the body made active again procreation is actually viable even making fully active body is possible given enough time and resources invested.

What's your lore source?

> in the game (RTS warcraft 2 or WOW) there is something called ankh of resurrection supposedly bring back the wielder after death so in theory there must be something that can reverse this undead status it maybe lost relic or very powerful item of light, ashbringer also originally from shadow tainted crystal in the end it just the game lore decide to not explore this option in depth because everyone busy waging war.

We don't see any examples in the lore. Moreover... there was one example, but Blizzard retconned it because it was against many lore rules and made concept of undead worthless.
https://wow.gamepedia.com/Ras_Frostwhisper

> the thing is in ck2 we are basically in open world with many possibility the drive to explore and achieve something despite the odds against it is the main drive to push forward in the game. like immortality quest it is not even real in real life or even proven historical fact that there is immortal people yet they (paradox) add it to the game obviously giving the game some flavor of what if scenario.

As I said above, the impossibility of returning back to the living is a core of the lore and the plot, so it breaks the motivation of main characters and their storylines and Blizzard understand it.

**Comment by virsia** [comment]
One race is still an example that demon actually capable in breeding, at this point we could argue that they just choose not to do it, most demon see other being as lesser being meant to be dominated.

The link you provide about Homunculi is the lore source of my argument, Demon hunter unable to detect homunculi or differentiate them with normal human.

Then again I did say what if scenario, it only add choice and something to achieve in the game because adoption seems rather easy and dull.

**Comment by zumbak04** [comment]
> One race is still an example that demon actually capable in breeding, at this point we could argue that they just choose not to do it, most demon see other being as lesser being meant to be dominated.

Yep, but it's one race. The exception that proves the rule.

> The link you provide about Homunculi is the lore source of my argument, Demon hunter unable to detect homunculi or differentiate them with normal human.

Yep, but they're still undead.

> Homunculi speak in coherent thoughts, and their skin is well-preserved. However, when sliced open a green liquid pours out of their bodies.

Like all the other undead.

> Then again I did say what if scenario, it only add choice and something to achieve in the game because adoption seems rather easy and dull.

Well, that's core thing. The only way to stop being undead is death. This is the main "engine" of the undead and the Forsaken plots. As well as becoming a demon... there's no way back. It's price you pay... like Illidan did.

**Comment by virsia** [comment]
if that so there is actually less interesting point of playing as demon or undead or even adding mechanics of changing religion if they are kind are somewhat useless, having bared the right to marry you cannot get titles by marriage, no eugenics since adopted children are predetermined, no pregnancy event or raising children event while this is the main core of some dlc there is also no chance of having child of destiny or other interesting things that in the game. 

it's not like I cannot solve this problem personally since what it takes for this to happen is just changing few lines add some new traits in the mod but still it's disappointing since in ck 2 it is house that matters and by doing that normally you need marriage mechanics and do eugenics in grooming that perfect ruler.

**Comment by geffe1** [comment]
The point of becoming a demon or Undead is to be immortal.. sort of. And you still get events to adopt other characters into your dynasty.

This however doesn't mean that you would get claims to a province or region, which to me also makes sense. You are after all not gonna have children and thus wouldn't be able to gain new titles that way. And I do not believe demons not undead care about marrying a human to get Stormwind under their "protection". They would simply conquer it.

**Comment by virsia** [comment]
> The point of becoming a demon or Undead is to be immortal.. sort of. And you still get events to adopt other characters into your dynasty.
> 
> This however doesn't mean that you would get claims to a province or region, which to me also makes sense. You are after all not gonna have children and thus wouldn't be able to gain new titles that way. And I do not believe demons not undead care about marrying a human to get Stormwind under their "protection". They would simply conquer it.

there is plenty way to gain immortality like starting as an elf, as a mortal there is immortality event chain that does not require you having -40 opinion penalty and having -1000% fertility.

not all demon are about brawn and brute force some employ plots and dwell in intrigue some seduce while other might just steamroll everything, staring as demon count tier even though you are legendary class you cannot defeat army 10 times of your own, yes in the lore Demon are powerful being capable raining brimstone from the sky on their enemies even tens of thousand soldier does not stand a chance but in ck2 you does not have that luxury.

**Comment by DJScias** [comment]
Perhaps the best way to tackle this is by adding more flavor content to undead and demon characters?

I'm sure events that are basically "abduct" or "manipulate" living creatures to then kill and raise (undead) or corrupt (demon) can be a thing.

However the main blocker for this, most likely, is finding someone who wants to make all these events and such.

**Comment by virsia** [comment]
well there is easy way and there is a hard way to do this, 

> Perhaps the best way to tackle this is by adding more flavor content to undead and demon characters?
> 
> I'm sure events that are basically "abduct" or "manipulate" living creatures to then kill and raise (undead) or corrupt (demon) can be a thing.
> 
> However the main blocker for this, most likely, is finding someone who wants to make all these events and such.

this is why I open this discussion so author can identify the problem, community is open to leave their ideas on how to improve the mod.

personally there is easier way to solve the problem without much tinkering on the base mod is just add fertility counter just name it blessing of light or sort upon changing religion this way demon or undead can marry although as far as I know 50-50 chance the child born died of poor health because they born as demon or born as undead there is something in the mod that automatically cull demonic culture or undead culture baby.

In this mod author make that the one that determine recruitment race is not the culture ( cul ) you follow but ethnicity ( g_cul ) so for example you are eredar ethnicity (g_cul) light religion courtier you spawn or recruit is eredar light religion. however even though courtier follow light religion the courtier also still carry demon trait since this also being determined in the text that eredar have demonic trait always, the random trait assigned is solely depend on what type religion you follow, good religion group will have good trait like kind, chaste, temperate and patient while evil religion group will almost certainly have cruel, wroth, greedy and anything that will displease people.

**Comment by zumbak04** [comment]
> if that so there is actually less interesting point of playing as demon or undead or even adding mechanics of changing religion if they are kind are somewhat useless, having bared the right to marry you cannot get titles by marriage, no eugenics since adopted children traits are predetermined, no pregnancy event or raising children event while this is the main core of some dlc there is also no chance of having child of destiny or other interesting things that in the game.

There's no point in playing immortal (event elf) if you like to play this way. You have to die 
so that your heir inherits titles.

> playing as a demon or undead mainly for roleplay purpose maybe experiencing new event and flavor and that role inculding how about we play as light religion demon oh yes since you are demon your child always be demon even though you are ruling lordaeronian county and your liege always hate you there is simply no way getting out of that or in anyway to redeem yourself, that's kinda unfair.

I suggested you the solution that absolutely fits the lore in my first message.

> not all demon are about brawn and brute force maybe arcdemon would do this but some especially weaker demon employ plots and dwell in intrigue some seduce while other might just steamroll everything

True, that's why we made Demonic Legion government elective.

> I'm sure events that are basically "abduct" or "manipulate" living creatures to then kill and raise (undead) or corrupt (demon) can be a thing.

It's already a thing in all the evil societies.

> personally there is easier way to solve the problem without much tinkering on the base mod is just add fertility trait +1000% counter just name it blessing of light or sort upon changing religion this way demon or undead can intermarry with other ethnicity although as far as I know 50-50 chance the child born died of poor health because they born as demon or born as undead there is something in the mod that automatically cull demonic culture or undead culture baby.

I think I said everything I could already.

> In this mod author make that the one that determine recruitment race is not the culture ( cul ) you follow but ethnicity ( g_cul ) so for example you are eredar ethnicity (g_cul) light religion even though you assimilate to local lordaeron for example courtier you spawn or recruit is eredar light religion. however even though courtier follow light religion the courtier also still carry demon trait since this also being determined in the text that eredar have demonic trait always, the random trait assigned is solely depend on what type religion you follow

Cho'gall will summon... whom? Orcs? Demons? You suggest to summon courtiers that will hate you... because of race or being (Undead, Demon, Void Being) trait. Moreover, for characters like Cho'gall it's hard to find ogre spouse.

> good religion group will have good trait like kind, chaste, temperate and patient while evil religion group will almost certainly have cruel, wroth, greedy and anything that will displease people.

It's already a thing.

**Comment by virsia** [comment]
uhhh even immortal character can have this flavor event, who said player need to die ? prerequisite in having pregnancy event flavor like craving and being comforted is just be pregnant and prerequisite in having child event as a parents obviously just get some children and act as their guardian on the contrary you cannot trigger or get this event if you are not parents or you die since you will play as heir you experiencing childside event not parent side event.

when you are a demon and not under legion command you are not having demonic legion government, you have usual feudal monarchy.

yeah adopt living heir........it just weird well you are the author up to you how you roll it just does not feels right, I can understand about undead having green goo inside cannot reproduce before they are alive again but demon like eredar they are biological body they are breathing and alive they used to be draenei if draenei made compatible to breed why not eredar, eredar that lived millennia full of brutality get bored and decide to swing other way? more original story.

then again about the lore why making everything so random with elaborate event branching giving many alternative while author seems determine to follow lore to the letter? 

I'm not suggesting summon courtier that hate you it just I see what inside your mod and I understand the mechanics, however while it is convenience to have courtier that doesn't appear to hate you in plain sight it is not in a way real, demon hate each other and lunatic cruel arbitrary courtier also will try to murder you or steal your treasury or even worse in my opinion hateful courtier does not change the game that much it's part of the story.

Cho'gall can marry other fel creature orks or whatever even captured prisoner if player is in control the offspring can be ogre or other race still 50-50 chance Cho'gall heir could be elf if player wants if author allow and player set game rules to be free intermingling half breed, the thing is other fel creature cannot or having luxury to do so they cannot enjoy this feature.

I don't mind doing stupid thing and getting myself killed as long I can continue the game normally, immortality is easy to get one immortal dies thousand still exist, the idea that some being cannot pick focus because being barred and some feature cannot be accessed that's not fun, player want to explore and trigger event the mods have to offer while they do their thing, personally I'm not interested playing as "historic" character they can do their own thing it's fine, we forge our own way, making our own legend and living the story of our own, many of us are fans of certain character in original game in a way we want to prevent misfortune to befall them or saving them from their fate we as fans want to help change their fate isn't this also what mods promise? 

the mod talk about something to make your character truly original doesn't it? it's in the mod page? having kind demon legendary mage marry lordaeron paladin guy having human child? how about free undead that breaks lich king control? player not desire to be gloom and doom like sylvanas, that's true original story not barred by unfinished lore of original game since we know WOW constantly update their lore there is plenty lost relic and ruins to explore not mention naga ruins in the abyss of the ocean maybe one of them have such treasure? have lost relic of forgotten age? we certaitly dont know for sure, do we wait for lore to offer us such story how competent is blizzard tbh or we decide to make our own additional lore? even sunwell can turn to a girl how is that even possible a well have consciousness and this is make sense while undead want to make artificial body to transfer his/her soul into is not possible or  not making any sense? Azeroth is world full of possibility, if ner'zhul who used to be ork lose his body and his soul can be transferred to iced armor on the frozen throne what makes undead soul cannot be transferred into artificial body? just because it is not being explored in the lore?

sometimes we get turn into something we don't want to like getting plague fighting for the good guys wham insta undead your culture instantly flip, you or your heir captured and lich king decide to enslave him/her/you, also you can turn to demon because fel corruption mechanics 100 fel corruption and you become demon and etc etc, the point is things happen and there is no way to redeem yourself.

the idea of adopted child you cannot change their way unlike children normally born you can change focus heritage or religion to match guardian, adopted will be adult unless author decide you can adopt babies which make more sense since you normally as parents does not want your child to know that he or she being adopted, when your child turn 12 in mod normally you assign training for class in this case demon or undead cannot have this event fired your heir pretty much just plain classless commoner even though you are legendary mage, you also cannot mold or control what this heir gonna be what identity is he/she, adopted heir automatically copy your ethnicity and your exact culture meaning once eredar your heir always will be eredar. he or she still be demon unless this mechanics also change by author for example good religion eredar demon have adopted draenei child as draenei is eredar non demonic counterparts. now we can get rid the demonic parts.

**Comment by zumbak04** [comment]
Dude, I'm sorry, but I'm not going to read it. It's... wall of text. The suggestion should be as short as possible. I don't even see what you quote or your lore sources.

I suggested you the only lore way I see. We don't plan to break the lore rules. If we do, it's not Warcraft.

**Comment by virsia** [comment]
well if author decide to stick to the lore, the question is why make event random or provide alternative outcome? this also considered breaking the lore.

**Comment by zumbak04** [comment]
> well if author decide to stick to the lore, the question is why make event random or provide alternative outcome? this also considered breaking the lore.

Then the alternative world from the WoD breaks the lore much more then anything else. Lore rules and storyline are different things.

**Comment by virsia** [comment]
that role rules preventing us from deciding some character fate or even changing their fate which in the feature. 
lore rules and original story are connected story makes the lore, if Golum never found the ring there will be no fellowship of the ring. if sauron decide to be good boy and not being rebellious maia there will be no lore at all lord of the ring story will not exist no battle of five army and there will be entirely different things going.

**Comment by zumbak04** [comment]
> that lore rules preventing us from deciding some character fate or even changing their fate which in the feature.

Nani?! Of course they're. It's what makes Wacraft... Warcraft. Using the same argument you can demand giving Sylvanas 100 Martail because it's preventing you from deciding her fate.

**Comment by virsia** [comment]
obviously not since sylvanas is not even ruler she is courtier here. does not matter if she has 100 all stats even you make her count with full 7 holding fully upgraded she still can be killed or captured or executed or abducted, the thing is there is so many choice about that but being barred and deprived of choice is no fun at all.

**Comment by virsia** [comment]
> Crusader Kings 2 got historical events, however you can go out of your way, which the AI will also be doing to change history. You can still try to walk a historical accurate path but it is unlikely that will ever happen as it is a sandbox game. Just like our vision for this mod is, to make as little railroading as possible.
> 
> As Arthas you can choose not to pick up Frostmourne.

yessss this is what I want CHOICE. I can choose to breed or not breed how is that hard? worried game get lag? set this decision for player only, we are not following the lore we make own lore here, why so stuck up, why no what if scenario? that's the problem here.

**Comment by zumbak04** [comment]
> yessss this is what I want CHOICE. I can choose to breed or not breed how is that hard? worried game get lag? set this decision for player only, we are not following the lore we make own lore here, why so stuck up, why no what if scenario? that's the problem here.

So you want game rule? Because other people may not like to play in the world where undead can breed.

**Comment by virsia** [comment]
well this is decision author can set it like on off toggle, so player can choose to incorporate this or not, or maybe even better make chain event to get rid undead trait so you are not undead, there is sense of accomplishment for achieving it, we want challenge not easy way out like adoption make it hard like immortality event I don't care, the harder the better for me personally then again as long there is a CHANCE.

the harder it is the more I strive for it CK2 have nothing interesting when you are emperor the fun is when you are count climbing up top.

**Comment by geffe1** [comment]
It is not possible for dead things to breed though. 

It is not about choice in that matter, it is just not possible in the WoW lore which we try to follow where we can. It would not be an intellectual honesty to allow them breeding  hence we got the adoption thing going on.

**Comment by virsia** [comment]
err I I'm not really into necrophilia here I understand that green goo inside undead, and not just undead the problem here, Demon also have problem can't marry and having normal heir, the fun of mixing race is not there no heir grooming event and barred from choosing 2 focus family and seduction.

**Comment by zumbak04** [comment]
> well this is decision author can set it like on off toggle, so player can choose to incorporate this or not, or maybe even better make chain event to get rid undead trait so you are not undead, there is sense of accomplishment for achieving it, we want challenge not easy way out like adoption make it hard like immortality event I don't care, the harder the better for me personally then again as long there is a CHANCE.

I doubt someone will make non-lore event chain, so better suggest something more simple that doesn't require a lot of work. When you come up with idea, just edit your first comment. Better mention only what you want and why. As short as possible.

**Comment by zumbak04** [comment]
@virsia 
P.S. And mention that it's game rule.

**Comment by virsia** [comment]
> I doubt someone will make non-lore event chain, so better suggest something more simple that doesn't require a lot of work. When you come up with idea, just edit your first comment. Better mention only what you want and why. As short as possible.

well like I mentioned above this can be achieved by several way honestly, make trait counter that infertility for demons and maybe touched by light so undead person that change religion to good group turn back to normal human being. 

adding if religion fel group add trait demon or if good group remove trait demon, changing few lines will do honestly. I dunno I don't want to dictate author does thing, but I don't like the idea of adoption like seriously maybe for undead it's fine too much hassle adoption will do but Demon should work just fine in breeding with creature of azeroth they are living creature just off this world (Azeroth) why not make Demon like void being Naga, Naga could breed honestly despite scale and and fins.

**Comment by zumbak04** [comment]
> Demon should work just fine in breeding with creature of azeroth they are living creature just off this world (Azeroth) why not make Demon like void being Naga, Naga could breed honestly despite scale and and fins.

Naga can breed because it's proved in the lore. Demons can't. Let's close this conversation already.

**Comment by virsia** [comment]
> > Demon should work just fine in breeding with creature of azeroth they are living creature just off this world (Azeroth) why not make Demon like void being Naga, Naga could breed honestly despite scale and and fins.
> 
> Naga can breed because it's proved in the lore. Demons can't. Let's close this conversation already.

Fine up to author if you refuse the idea I can respect that, but I seriously cannot accept this I'll make my own patch for this modify the files as I like for my personal use. It's just sad this not being explored it have so many potential for interesting story tbh.

**Comment by zumbak04** [comment]
> Fine up to author if you refuse the idea I can respect that, but I seriously cannot accept this I'll make my own patch for this modify the files as I like for my personal use. It's just sad this not being explored it have so many potential for interesting story tbh.

REEE! Are you trolling?!

> So you want game rule? Because other people may not like to play in the world where undead can breed.

> I doubt someone will make non-lore event chain, so better suggest something more simple that doesn't require a lot of work. When you come up with idea, just edit your first comment. Better mention only what you want and why. As short as possible.

**Comment by virsia** [comment]
No, Im not
I am capable modifying mods or making own mods to rectify some situation, I'm dead serious did you not read above I suggest you make trait to counter that ridiculous -1000% fertility.

believe me when I say if you are undead you can give birth to human or any thing 50-50% chance but when your child is undead gfx she/he will die poor health. I don't want to change a lot of what intended but seems I have no choice. to make trait all you need to do is making trait in common trait, make custom gfx and put in gfx and make some localization. done in 10 15 mins, and no I dont make any story I just modify the decision when changing religion if root have trait demon add trait  XXX. if rood have trait undead remove trait undead. not proper patch but patch that works the story is in my head.

**Comment by DukeJeans** [comment]
In essence,
virsia said:
> well this is decision author can set it like on off toggle, so player can choose to incorporate this or not

zumbak04 said:
> So you want game rule? Because other people may not like to play in the world where undead can breed.

This is artificial conflict at this point, closing the issue because _obvious reasons._
